tools/depends/target/Makefile

```diff
@@ -186,6 +187,7 @@ taglib: $(ZLIB)
 wayland: expat libffi
 waylandpp: wayland $(WAYLANDPP_DEPS)
 xz: gettext
+exiv2: $(ICONV) $(ZLIB) expat
```

Comment:
alphabetical order please

Reply:
Still a rebase issue here. Its correct in "[depends] Add exiv2 to depends" commit, then this line is added back in "Pictures: Move to exiv2 for image metadata retrieval (exif and iptc)"

Reply:
fixed, thanks